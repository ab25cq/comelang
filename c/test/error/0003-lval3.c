/*
PATTERN: c:10
PATTERN: lvalue
*/
int
main()
{
	int *p;	
	
	p = &2;
	return 0;
}
