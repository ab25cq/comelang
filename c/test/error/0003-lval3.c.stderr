& expects an lvalue at test/error/0003-lval3.c:10:10
    p = &2;
         ^
